include/exadg/functions_and_boundary_conditions/evaluate_functions.h

```diff
@@ -248,7 +246,7 @@ struct FunctionEvaluator<2, dim, Number>
 {
   static inline DEAL_II_ALWAYS_INLINE //
     dealii::Tensor<2, dim, dealii::VectorizedArray<Number>>
-    value(std::shared_ptr<dealii::Function<dim>> const                function,
+    value(dealii::Function<dim> &                                     function,
```

Comment:
One could also write:
```suggestion
    value(std::shared_ptr<dealii::Function<dim>> const &                                   function,
```

Reply:
Yes but why would we pass in a pointer if we're not doing anything with the pointer but just need the object it points at, right?